.travis.yml

```diff
@@ -3,6 +3,10 @@ matrix:
   include:
     - python: 3.6
       env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: true
```

Comment:
Why `sudo: true` here? Also, why `dist: xenial` only for Python 3.7 instead of globally?

Reply:
It looks like this is the configuration required for python 3.7: https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905

I can set this configuration to global if you want.